Automattic/pocket-casts-android
# EOY 2023: Intro Story

| 🎨 Designs: lH66LwxxgG8btQ8NrM0ldx-fi-1599_20385 |
|:---:|

## Description

This updates the visuals for 2023 intro story.

## Testing Instructions

** Test on a real device so you can test the parallax effect.

1. Make sure you're logged in into an account that has a few episodes listened
2. Go to `Profile` > `Settings` > `Beta Features` > enable `End of Year`
3. Go back to `Profile`
4. Tap the EOY image
5. ✅ Check that the Intro Story looks good
6. Tilt your device
7. ✅ The "2023" should move around

## Screenshots or Screencast 

https://github.com/Automattic/pocket-casts-android/assets/1405144/15155d24-d73b-46a4-aef0-f14a040acb7c

Logo color - yet to be updated

## Checklist
 
#### I have tested any UI changes...

- [ ] for accessibility with TalkBack <- this is not working, I need to look into it.
